server/neptune/workflows/activities/github.go

```diff
@@ -316,7 +317,22 @@ type ListModifiedFilesResponse struct {
 }
 
 func (a *githubActivities) GithubListModifiedFiles(ctx context.Context, request ListModifiedFilesRequest) (ListModifiedFilesResponse, error) {
-	// TODO: Use client.ListModifiedFiles(ctx, owner, repo, pullNumber) to list files modified in a PR
-	// internal.Repo object and internal.PullRequest has all the necessary fields to make this call
-	return ListModifiedFilesResponse{}, nil
+	files, err := a.Client.ListModifiedFiles(
+		internal.ContextWithInstallationToken(ctx, request.Repo.Credentials.InstallationToken),
+		request.Repo.Owner,
+		request.Repo.Name,
+		request.PullRequest.Number,
+	)
+	if err != nil {
+		return ListModifiedFilesResponse{}, errors.Wrap(err, "listing modified files in pr")
+	}
+
+	filepaths := []string{}
+	for _, file := range files {
+		filepaths = append(filepaths, file.GetFilename())
+	}
+
+	return ListModifiedFilesResponse{
+		FilePaths: filepaths,
```

Comment:
Have you thought about response size here? I'd like a comment in the code that addresses it and gives some estimation on when this would break.

Reply:
Yup, I had it above `ListModifiedFilesResponse` struct declaration but I've moved it to where we return the response to make it clearer.